tests/dragonfly/snapshot_test.py

```diff
@@ -46,6 +51,7 @@ async def test_consistency(df_factory, format: str, seeder_opts: dict):
     Test consistency over a large variety of data with different sizes
     """
     dbfilename = f"dump_{tmp_file_name()}"
+    rm_glob(f"{dbfilename}*")
```

Comment:
I think there is a better way to do, somewhere in conftest.py upon a test shutdown. For example inside `df_factory` fixture after the yield statement. @dranikpg  what do you think?

Reply:
I was thinking the same as well, we just need to be careful because some tests might use `Stop` explicitly, we should not remove those files as part of that because it might break the semantics of some of the tests.